src/applications/check-in/utils/selectors/feature-toggles.js

```diff
@@ -15,10 +15,12 @@ const selectFeatureToggles = createSelector(
     isEmergencyContactEnabled: toggleValues(state)[
       FEATURE_FLAG_NAMES.checkInExperienceEmergencyContactEnabled
     ],
-
     isUpdatePageEnabled: toggleValues(state)[
       FEATURE_FLAG_NAMES.checkInExperienceUpdateInformationPageEnabled
     ],
+    isPaymentAndDebtSectionEnabled: toggleValues(state)[
```

Comment:
Are you using this team's selectors?

Reply:
Yes I am. Trying to limit duplicate code, however if this is not how it should be working then I can make my own generateFeatureToggles. I assumed duplicate code like that was frowned upon. Still new to the size of this organization. My old company had 4 developers.